main.tf

```diff
@@ -28,9 +32,161 @@ resource "ibm_app_config_collection" "collections" {
   tags          = each.value.tags
 }
 
+##############################################################################
+# Configuration aggregator
+##############################################################################
+
+# Create the required Trusted Profile
+module "config_aggregator_trusted_profile" {
+  count                       = var.enable_config_aggregator ? 1 : 0
```

Comment:
Should this still be created when the enterprise trusted profile is created, or should we add the `var.config_aggregator_enterprise_id` as a part of the condition?

Reply:
This is always needed for both normal and enterprise use case